spec/models/profile_spec.rb

```diff
@@ -264,14 +264,16 @@
     end
 
     it 'does not activate a profile if under fraud review' do
-      profile.update(fraud_review_pending_at: 1.day.ago)
```

Comment:
I don't think these date values are significant, only that they make the column not null.